src/twisted/web/test/test_cgi.py

```diff
@@ -87,8 +87,8 @@
 """
 
 URL_PARAMETER_CGI = """\
-import cgi
-fs = cgi.FieldStorage()
+from twisted.web import _cgi
+fs = _cgi.FieldStorage()
```

Comment:
If you take my suggestion above I think that we can avoid vendoring `FieldStorage` entirely by replace this with something that echoes `os.environ['QUERY_STRING']` ([RFC](https://www.rfc-editor.org/rfc/rfc3875#section-4.1.7)). `FieldStorage` [appears to have been used purely for convenience](https://github.com/twisted/twisted/commit/e90bfd4c09da8e6168e2b4c711099a8ce927a675), and as the `cgi` module is going away I don't think there's any sense verifying compatibility with it.

`HEADER_OUTPUT_CGI` looks like it'd work for this.